pkg/distro/rhel9/azure.go

```diff
@@ -414,7 +414,7 @@ func azureRhuiBasePartitionTables(t *imageType) (disk.PartitionTable, bool) {
 	}
 }
 
-var defaultAzureKernelOptions = "ro console=tty1 console=ttyS0 earlyprintk=ttyS0 rootdelay=300"
+var defaultAzureKernelOptions = "ro loglevel=3 console=tty1 console=ttyS0 earlyprintk=ttyS0 rootdelay=300"
```

Comment:
I'm not 100% sure if adding `loglevel=3` is a good idea, but it is used in RHEL documentation and RHEL images built by MSFT in their marketplace have it set... :thinking: